### PR TITLE
Changes to api key method

### PR DESCRIPTION
**Pull request by neekssss**
Moved api key to NewsServlet.java in preparation for potential storage of key on server. Also modified method of list creation for the news list (Now uses forEach() method instead of for loop). 

ISSUE: In making these changes, I came across an issue where when you search for news, the results get shown multiple times based on the number of previous news searches(i.e. If I had searched for New York and Los Angeles news prior to Denver, the results for Denver would show 3 times).

TL;DR: News api method cleaned up, significant bug made apparent as a result. Preparations made for more secure api key storage.

NewsBug

### CODE REVIEW

**Review by jasmine-r — COMMENTED** [review]
Changes look good! 
We should probably standby for approvals on PR 7 and 8.
If PR 7 and 8 are approved and merged into master, we may need to resolve potential merge conflicts on PR 9

**Comment by neekssss** [comment]
> Changes look good!
> We should probably standby for approvals on PR 7 and 8.
> If PR 7 and 8 are approved and merged into master, we may need to resolve potential merge conflicts on PR 9

I agree, while the current version of the news api has this same bug here, the PR 9 code exacerbates it.
I just thought to put the pull request up just so we have record of it.

**Comment by neekssss** [comment]
Issue resolved, code ready for merge. Pending review.

**Review by Zoe9711 — APPROVED** [review]

**Review by jasmine-r — APPROVED** [review]
